.github/workflows/test.yml

```diff
@@ -145,3 +145,60 @@ jobs:
         env:
           DEVELOCITY_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
         run: ci/scripts/java_test.sh $(pwd) $(pwd)/build
+
+  integration:
+    name: AMD64 integration
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Checkout Arrow
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: 0
+          repository: apache/arrow
+          submodules: recursive
+      - name: Checkout Arrow Rust
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: apache/arrow-rs
+          path: rust
+      - name: Checkout Arrow nanoarrow
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: apache/arrow-nanoarrow
+          path: nanoarrow
+      - name: Checkout Arrow Go
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: apache/arrow-go
+          path: go
+      - name: Checkout Arrow Java
```

Comment:
We don't need `repository` here?

Reply:
We don't need `repository` here because the default `repository` is `apache/arrow-java` here.
If we specify `apache/arrow-java` explicitly, we can't run integration test in fork (e.g. `kou/arrow-java`).